ddtrace/internal/writer.py

```diff
@@ -111,7 +111,7 @@ def get_json(self):
                 # This typically happens when using a priority-sampling enabled
                 # library with an outdated agent. It still works, but priority sampling
                 # will probably send too many traces, so the next step is to upgrade agent.
-                log.debug("Cannot parse Datadog Agent response, please make sure your Datadog Agent is up to date")
+                log.debug("Cannot parse Datadog Agent response. This occurs because Datadog agent is out of date or DATADOG_PRIORITY_SAMPLING=false is set")
```

Comment:
I'd like to avoid mentioning `DATADOG_PRIORITY_SAMPLING=false` if we can

I am also wondering if we can make any changes to this logic to detect a scenario that would cause an issue?

https://github.com/DataDog/dd-trace-py/blob/7f123cdb813baa133de380f57cfae60406a532f2/ddtrace/internal/writer.py#L345

cc @Kyle-Verhoog or @majorgreys what do you think for this?

Reply:
What's the reasoning for not mentioning it? It's in our docs: https://docs.datadoghq.com/tracing/setup_overview/setup/python/?tab=containers#configuration
Should we remove it from there? 
Would a more general: "This occurs because Datadog agent is out of date or sampling priority is set to false" work?

Reply:
yeah, that should be removed from our docs. no one should have that set (in fact we should probably remove the configuration... I think if we can)

Reply:
Made this PR: https://github.com/DataDog/documentation/pull/10422

Reply:
awesome! thanks for doing that.